resources/views/components/movie-card.blade.php

```diff
@@ -0,0 +1,29 @@
+@props([
+    "title" => "",
+    "year" => "",
+    "length" => "",
+    "description" => "",
+    "movie" => [],
```

Comment:
```suggestion
    "title",
    "year",
    "length",
    "description",
    "movie",
```
None of these are optional, they should all be required when calling for this component

Reply:
thats the point, they should always contain the props

Reply:
fixed it